sensor/common/registry/registry_store.go

```diff
@@ -28,8 +28,9 @@ import (
 const (
 	defaultSA = "default"
 
-	pullSecretNamePrefix = "PullSec"
-	globalRegNamePrefix  = "Global"
+	PullSecretNamePrefix = "PullSecret"
+	GlobalRegNamePrefix  = "GlobalPullSecret"
+	NoAuthNamePrefix     = "NoAuth"
```

Comment:
nit: Move this to `scan.go` since not used by the registry store. FWIW - I could see the argument for wanting to 'centralize' these prefixes as well.

Reply:
I centralized the name prefixes by putting them into `pkg/registries/types`.